plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/N4JSFlowAnalyses.java

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) 2017 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.flowgraphs;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.TreeSet;
+
+import org.eclipse.n4js.flowgraphs.analyses.DirectPathAnalyses;
+import org.eclipse.n4js.flowgraphs.analyses.GraphVisitorAnalysis;
+import org.eclipse.n4js.flowgraphs.analyses.GraphVisitorInternal;
+import org.eclipse.n4js.flowgraphs.analyses.SuccessorPredecessorAnalysis;
+import org.eclipse.n4js.flowgraphs.factories.ControlFlowGraphFactory;
+import org.eclipse.n4js.flowgraphs.model.FlowGraph;
+import org.eclipse.n4js.n4JS.Block;
+import org.eclipse.n4js.n4JS.CatchBlock;
+import org.eclipse.n4js.n4JS.ControlFlowElement;
+import org.eclipse.n4js.n4JS.Script;
+
+import com.google.common.collect.Lists;
+
+/**
+ *
+ */
+public class N4JSFlowAnalyses {
+	private FlowGraph cfg;
+	private DirectPathAnalyses dpa;
+	private GraphVisitorAnalysis gwa;
```

Comment:
* since `dpa` is the only field here that holds several analyses, why not call it `dpas`?
* why `gwa` instead of `gva` (since it holds a ref to a  "graph Visitor analysis")?

Reply:
It feels like I rename everything. BTW: GraphVisitor was once called GraphWalker, hence the variable name `gwa`.